frontend/packages/ux-editor/src/utils/datamodel.ts

```diff
@@ -67,3 +68,25 @@ export const getXsdDataTypeFromDataModel = (
 
   return element?.xsdValueType;
 };
+
+const generalFilter = (element: DatamodelFieldElement) =>
+  element.dataBindingName && element.maxOccurs <= 1;
+const repeatingGroupFilter = (element: DatamodelFieldElement) =>
+  element.dataBindingName && element.maxOccurs > 1;
+const multipleAttachmentsFilter = (element: DatamodelFieldElement) =>
+  element.dataBindingName && element.maxOccurs > 1 && element.xsdValueType === 'String';
+
+export const dataModelFieldsFilter = (
```

Comment:
Would prefix the name here with `get` to make it clear that the function returns a filter
```suggestion
export const getDataModelFieldsFilter = (
```